python/paddle/cost_model/cost_model.py

```diff
@@ -53,17 +54,33 @@ def profile_measure(self,
         exe.run(startup_program)
         paddle.fluid.profiler.start_profiler("All")
         exe.run(main_program, feed={"X": x}, fetch_list=[])
-        # core.CostModel.ProfileMeasure(main_program, device)
-        print("core:<<<<<<<")
 
         cost_model = core.CostModel()
         cost_data = cost_model.ProfileMeasure(device)
-        # cost_list = self.stop_cost_model()
-        # return cost_list
 
+    def static_cost_data(self):
+        static_cost_data_path = os.path.join(
+            os.path.dirname(__file__), "static_op_benchmark.json")
+        with open(static_cost_data_path, 'r') as load_f:
+            load_dict = json.load(load_f)
+        self._static_cost_data = load_dict
+        # return all static cost data
+        return load_dict
 
-cost_model = CostModel()
+    def get_static_op_time(self, op_name, forward=True, dtype="float32"):
 
-startup_program, main_program = cost_model.build_program()
+        if op_name == None:
+            raise ValueError(
+                'op_name should not be empty when you want to get static op time'
+            )
 
-cost_model.profile_measure(startup_program, main_program)
+        op_cost = {}
+        for op_data in self._static_cost_data:
+            if (op_data["op"] == op_name) and (dtype in op_data["config"]):
+                if (forward == True):
```

Comment:
Clean the code:

`if forward:`

is enough

Reply:
Done.